src/test/scala/is/hail/methods/LDPruneSuite.scala

```diff
@@ -110,7 +110,7 @@ object LDPruneSuite {
 class LDPruneSuite extends SparkSuite {
   val bytesPerCoreMB = 256
   val nCores = 4
-  val vds = TestUtils.splitMultiHTS(hc.importVCF("src/test/resources/sample.vcf.bgz", nPartitions = Option(10)))
+  lazy val vds = TestUtils.splitMultiHTS(hc.importVCF("src/test/resources/sample.vcf.bgz", nPartitions = Option(10)))
```

Comment:
this was bothering me because it loaded the vcd regardless of whether the subset of test methods I was running used the vds.

Reply:
Neat!